syntax = "proto3";

package project_manager;

service ProjectManager{
    rpc GetProjects(GetProjectRequest) returns (ProjectList) {}
}

message Credentials{
    string credentials = 1;
}

message Project{
    string project_id = 1;
    string name = 2;
}

message GetProjectRequest{
    Credentials credentials = 1;
}

message ProjectList{
    repeated Project projects = 1;
}